spec/system/admin/bulk_order_cancellation_spec.rb

```diff
@@ -29,14 +29,17 @@
       expect(page).to have_selector('span', text: 'COMPLETE', count: 2)
 
       page.check('selectAll')
-      page.find('.ofn-drop-down').click
-      page.find('.menu').find('span', text: 'Cancel Orders').click
+      page.find("span.icon-reorder", text: "ACTIONS").click
+      within ".ofn-drop-down-with-prepend .menu" do
+        page.find("span", text: "Cancel Orders").click
+      end
 
-      within '.modal' do
-        click_on "OK"
+      within '.reveal-modal' do
+        find_button("Confirm").click
       end
 
       # Verify that the orders have a STATE of CANCELLED
+      sleep(1)
```

Comment:
We tend to avoid this kind of `sleep` in our test. 
I tested, and if I remove the line, the test pass locally. 

We could also test the order model state changed.

That's why I suggest:

```suggestion
within '.reveal-modal' do
        expect {
          find_button("Confirm").click
        }.to change { o1.reload.state }.from('complete').to('canceled')
          .and change { o2.reload.state }.from('complete').to('canceled')
      end

      # Verify that the orders have a STATE of CANCELLED
      expect(page).to have_selector('span', text: 'CANCELLED', count: 2)
```